app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -0,0 +1,68 @@
+package org.fossasia.openevent.app.event.attendees;
+
+import android.content.Context;
+
+import org.fossasia.openevent.app.data.EventDataRepository;
+import org.fossasia.openevent.app.data.UtilModel;
+import org.fossasia.openevent.app.data.contract.IEventDataRepository;
+import org.fossasia.openevent.app.data.contract.IUtilModel;
+import org.fossasia.openevent.app.data.models.Attendee;
+import org.fossasia.openevent.app.event.attendees.contract.IAttendeesPresenter;
+import org.fossasia.openevent.app.event.attendees.contract.IAttendeesView;
+
+public class AttendeesPresenter implements IAttendeesPresenter {
+
+    IEventDataRepository eventDataRepository;
+    IAttendeesView attendeesView;
+    Long eventId;
+
+    public AttendeesPresenter(IAttendeesView attendeesView, Context context, Long eventId) {
+        this.attendeesView = attendeesView;
+        eventDataRepository = new EventDataRepository(new UtilModel(context));
+        this.eventId = eventId;
+    }
+
+    @Override
+    public void attach() {
+        loadAttendees(true);
+    }
+
+    @Override
+    public void dettach() {
+        attendeesView = null;
+    }
+
+    @Override
+    public void loadAttendees(boolean forceReload) {
+        attendeesView.showProgressBar(true);
+        eventDataRepository.getAttendees(eventId, forceReload)
+            .subscribe(attendees -> {
+                attendeesView.loadAttendeesIntoList(attendees);
+                attendeesView.showProgressBar(false);
+            }, throwable -> {
+                attendeesView.showProgressBar(false);
```

Comment:
Maintain a strict pattern of commands so that tests are very compatible with each other,

If you disable progressbar after results in success, you should disable it after error too